clients/javascript/lib/gen_types/CalendarDTO.ts

```diff
@@ -14,6 +14,8 @@ export type CalendarDTO = {
    * UUID of the user that owns the calendar
    */
   userId: ID
+  name: string | null
+  key: string | null
```

Comment:
I noticed these are documented in your Rust DTOs, but not coming through here. Is it because of `//` instead of `///`?